src/ophys_etl/workflows/db/schemas.py

```diff
@@ -203,6 +203,41 @@ def _generate_binary_mask(
             ] = True
         return mask.tolist()
 
+    def is_valid(self, equipment: str = "") -> bool:
+        """Return True if no processing flags (motion_border, decrosstalk)
+        flags are set.
+
+        Parameters
+        ----------
+        include_decrosstalk : bool Optional
+            Check decrosstalk flags for is_valid. Decrosstalk is required
+            for mesoscope experiments imaging multiple experiments at once.
+            Other experiments using different equipment do not require
+            decrosstalk.
+
+        Returns
+        -------
+        is_valid : bool
+            Composite flag that returns True if no processing flags are set.
+            False if any flag is set.
+        """
+        flag_set = [self.is_in_motion_border]
+        if equipment.startswith("MESO"):
+            flag_set += [self.is_decrosstalk_ghost,
+                         self.is_decrosstalk_invalid_raw_active,
+                         self.is_decrosstalk_invalid_raw,
+                         self.is_decrosstalk_invalid_unmixed,
+                         self.is_decrosstalk_invalid_unmixed_active]
+            for flag in flag_set:
```

Comment:
you need to separate out the decrosstalk flags so that this error message referencing "decrosstalk" doesn't apply to all flags, but only the decrosstalk flags